cottontaildb_client/cottontaildb_client.py

```diff
@@ -266,6 +266,21 @@ def get_entity_details(self, schema, entity):
         }
 
         return entity_details
+    
+    def preview_entity(self, schema, entity, limit=10):
```

Comment:
I'm not sure it makes sense to have a `preview_entity` function here, since the `cottontaildb_client` is only meant for core API functionality. A `preview` function strongly suggests only CLI usage and as a result I suggest to either extract this function to the `cottontaildb_cli` (which I would prefer) or to rename it to something like `sample_entity` to clear up its functionality for a more general use case as convenience function.